package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/Wolox/eslint-config-wolox-node#readme",
   "peerDependencies": {
-    "eslint": "5.9.0",
+    "eslint": "^6.4.0",
```

Comment:
Check comment of wolox eslint, applies same here, we should be less restrictive with peerDependencies, some like this:

```
    "eslint": "6.x",
    "eslint-config-wolox": "3.x",
    "prettier": "1.x",
    "prettier-eslint": "9.x",
    "eslint-plugin-prettier": "3.x",
    "eslint-plugin-import": "2.x"
```

Reply:
"eslint-config-wolox": "3.x", should be 4.x but it depends on the other one being released